fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -107,13 +108,25 @@
     String importPartitionResourceType;
 
 
+    /**
+     * If validate FHIR resources.
+     */
+    @Inject
+    @BatchProperty(name = Constants.IMPORT_FHIR_IS_VALIDATION_ON)
+    String fhirValidation;
+
+
     public ChunkWriter() {
         super();
     }
 
     @Override
     @SuppressWarnings("unchecked")
```

Comment:
what is unchecked below?

Reply:
List<Resource> fhirResourceList = (List<Resource>) objResJasonList;

Reply:
changed to suppress the line instead of the whole function

Reply:
sounds good, made a slight comment on JSON